pkg/minikube/config/types.go

```diff
@@ -37,6 +37,7 @@ type Config struct {
 
 // MachineConfig contains the parameters used to start a cluster.
 type MachineConfig struct {
+	Name                string
```

Comment:
add comment what is Name, Cluster name ? Profile name ? or this specific machine's unique name ?

Reply:
it's the machine's name, I just didn't want MachineConfig.MachineName to be a thing

Reply:
Since it's a public field. For anyone reading it.